docs/src/main/sphinx/admin/properties.rst

```diff
@@ -21,3 +21,80 @@ may be used to tune Trino or alter its behavior when required.
     Logging <properties-logging>
     Web UI <properties-web-interface>
     Regular expression function <properties-regexp-function>
+
+Property value types
```

Comment:
nit: for this entire entire section:

`Properties of a ``xxxxx`` type` -> `Properties of type ``xxxxxx`` ` reads better IMO. WDYT?

Reply:
Do we also want to mention that properties have validations defined on them - so any "ok" values that are out of range for a given property will read to validation errors?

Reply:
yes .. we should add info about min max and such

Reply:
Reworded, and added note about validation errors.